cmk/special_agents/agent_cisco_meraki.py

```diff
@@ -426,11 +508,10 @@ def agent_cisco_meraki_main(args: Args) -> int:
         hostname=args.hostname,
         section_names=args.sections,
     )
-
     sections = _query_meraki_objects(
         organisations=[
-            MerakiOrganisation(config, organisation)
-            for organisation in _get_organisations(config, args.orgs)
+            MerakiOrganisation(config=config, organisation=organisation)
```

Comment:
key-worded args are not necessary because config and organisation are distinguishable by default.